client/dwh_migration_client/batch_sql_translator.py

```diff
@@ -123,7 +123,8 @@ def start_translation(self) -> None:
 
         logging.info("The job finished successfully!")
         logging.info(
-            "To view the job details, please go to the link: %s", self._get_ui_link(job_name)
+            "To view the job details, please go to the link: %s",
+            self._get_ui_link(job_name),
```

Comment:
why do we need to add a comma here?

Reply:
https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html#:~:text=Black%20will%20add%20trailing%20commas,the%20allotted%20line%20length%20limit.

Reply:
Got it. It's a code formatting tool. Thanks.